rskj-core/src/main/java/co/rsk/rpc/Web3RskModule.java

```diff
@@ -47,4 +50,7 @@ default void rsk_flush() {
     }
 
     RskModule getRskModule();
+
+    String rsk_getStorageBytesAt(HexAddressParam address, HexNumberParam storageIdx, BlockRefParam blockRefParam);
```

Comment:
just to understand, is there a reason why the default keyword isn't declared?

Reply:
I'm only declaring the method and in this case, it is not needed to add default implementation. 

default methods are used when you want to implement a default method behavior within an interface to not break compatibility with older implementations. It is mainly used in high-exposure interfaces but here it is not needed.

Reply:
great, huge thanks angel.